lwgenerate/aws/aws.go

```diff
@@ -527,10 +527,10 @@ func createCloudtrail(args *GenerateAwsTfConfigurationArgs) (*hclwrite.Block, er
 			if args.SnsEncryptionEnabledSet {
 				if args.SnsEncryptionEnabled {
 					if args.SnsEncryptionKeyArn != "" {
-						attributes["sns_encryption_key_arn"] = args.SnsEncryptionKeyArn
+						attributes["sns_topic_encryption_key_arn"] = args.SnsEncryptionKeyArn
```

Comment:
Do we want to update the arg as well?
Will leave it up to you as it will work either way